src/bluetooth/VaccineDataAccess.js

```diff
@@ -46,11 +46,15 @@ export class VaccineDataAccess {
 
   upsertBreaches = breaches =>
     this.db.write(() => {
-      this.db.update(VACCINE_ENTITIES.TEMPERATURE_BREACH, breaches);
+      breaches.forEach(breach => {
```

Comment:
@katherine-sussol and @mark-prins (and @joshxg even though he helped diagnose haha) in case you miss it: we can't give `update` arrays, realm requires us to loop them. We could override the method on the database to check the type; if object/realm.object, do `update`, otherwise loop and `update`?

Reply:
Oops, this is my dodgy code. I ran into the same problem when trying to get the download thunk working haha none of the logs were making it to the DB 😿 . Updating the update method sounds like a nice way to abstract the problem away 👍

Reply:
NP! The ground work was laid, but hard to test without the stuff to test it. But with this PR we have some stuff!